index.js

```diff
@@ -3,7 +3,11 @@
     env: {
       jsSetBackgroundColor: function(valsPtr) {
         let vals = new Uint8ClampedArray(mod.exports.memory.buffer, valsPtr, 3)
-        let rgb = `rgb(${vals[0]}, ${vals[1]}, ${vals[2]})`
+        let paddedVals = ['', '', '']
```

Comment:
Are you familiar with `map`? You could call `vals.map` and pass it `padValue` as an argument, since `padValue` is a function that takes a value.

Reply:
Here's an [example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Mapping_an_array_of_numbers_to_an_array_of_square_roots) of passing map a function that takes a value

Reply:
I used to be familiar with `map`! Thanks for the code review, @quelledanielle; I'll update the PR accordingly.